content/md/imports-exports/automate-imports-and-exports.md

```diff
@@ -12,36 +12,75 @@ related: imports, exports, tailored-export, tailored-import,
 1. Click `Edit` in the top right corner
 1. Click `Properties`
-1. Under "Connection", select any choice from the dropdown list. For instance: `SFTP`
+1. Under "Type", select `SFTP`
 ![Select a connection type](../img/Automation_connection_type.png)
 1. Fill in all mandatory fields: `Filepath`, `Host`, `Port`, `Login` and `Password`
 ![Fill in all fields](../img/Automation_SFTP_configured.png)
 1. Click `Save` in the top right corner of the screen
 
-::: tips
-* If you want to automate your product exports, we support the patterns `%job_label%` & `%datetime%`. You can combine them to create a filepath like the following: "/myfolder1/mysubfolder2/export_%job_label%_%datetime%.xlsx".
-* You can use the `Test connection settings` button to check your settings and make sure your connection is valid.
-* If you want to authenticate your SFTP server, you can paste its fingerprint in the optional field `Host fingerprint`.
+:::tips
+If you want to authenticate your SFTP server, you can paste its fingerprint in the optional field `Host fingerprint`.  
 The accepted format of the fingerprint depends on the format of the server's public key: MD5 for `ssh-rsa` signatures, SHA-512 for others.
 :::
 
-:::info
-* You can only import media files with a .ZIP archive. Your spreadsheet should contain a filepath column.
-* If you export media files to a remote server, it will create a folder with the spreadsheet and the related media files. This folder won't be compressed (no .ZIP archive).
-:::
+## Connect to Amazon S3
 
-Please note that if you're importing from a remote server, the host you configured will be displayed.
-![Host displayed](../img/Automation_import_host_displayed.png)
+To connect your product import/export to Amazon S3:
+1. Go to `Imports` or `Exports`
+1. Select the profile you would like to automate
+1. Click `Edit` in the top right corner
+1. Click `Properties`
+1. Under "Type", select `Amazon S3`
+1. Fill in all mandatory fields: `Filepath`, `Connection string` and `Container name`
+1. Click `Save` in the top right corner of the screen
+
+## Connect to Microsoft Azure
+
+To connect your product import/export to Microsoft Azure:
+1. Go to `Imports` or `Exports`
+1. Select the profile you would like to automate
+1. Click `Edit` in the top right corner
+1. Click `Properties`
+1. Under "Type", select `Amazon S3`
+1. Fill in all mandatory fields: `Filepath`, `Region`, `Bucket name`, `Key` and `Secret`
+1. Click `Save` in the top right corner of the screen
+
+
+## Connect to Google Cloud Storage
+
+To connect your product import/export to Google Cloud Storage:
+1. Go to `Imports` or `Exports`
+1. Select the profile you would like to automate
+1. Click `Edit` in the top right corner
+1. Click `Properties`
+1. Under "Type", select `Amazon S3`
+1. Fill in all mandatory fields: `Filepath`, `Project ID`, `Service account` and `Bucket`
+1. Click `Save` in the top right corner of the screen
 
 # Schedule and automate your import/export profile
```

Comment:
I think we should add a point on the info block: 
- You cannot automate an import without remote storage

Reply:
Thanks, added!